core/src/message/mod.rs

```diff
@@ -78,6 +77,14 @@ impl Display for PayloadSizeError {
 /// Payload type for message.
 pub type Payload = LimitedVec<u8, PayloadSizeError, MAX_PAYLOAD_SIZE>;
 
+impl Payload {
+    /// Get payload length as u32.
+    pub fn len_u32(&self) -> u32 {
```

Comment:
NOTE: add this method in oder to be able to use payload len as u32 without panics in gear-core/src/context.rs